src/main/java/seedu/us/among/logic/parser/RunCommandParser.java

```diff
@@ -53,11 +55,13 @@ public RunCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RunCommand.MESSAGE_USAGE));
         }
 
-        Method method = ParserUtil.parseMethod(argMultimap.getValue(PREFIX_METHOD).get());
-        Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        Method method = ParserUtil.parseMethod(argMultimap.getValue(PREFIX_METHOD).orElse(""));
+        Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).orElse(""));
```

Comment:
why is orElse used, i think a assert before this statement is more apt since there always should be a method and address, hence the method will never be empty

Reply:
given that it is never empty, using .get() or orElse() makes no difference, so I think it is better to avoid get